.github/workflows/ci-weekly.yml

```diff
@@ -4,12 +4,9 @@ on:
   push: 
```

Comment:
the push and pull_request events should be reverted and placed in a different PR since they are unrelated to this work.  I believe you have another PR already for this or coming?

Reply:
Yes. I will have a PR for that.